app/components/HomeComponent/index.js

```diff
@@ -16,16 +16,18 @@ import {
 import InputModal from '../InputModal';
 import SecondaryNavComponent from '../SecondaryNavComponent';
 import OverviewComponent from './OverviewComponent';
+import { languagePluginLoader } from "../../utils/pyodide/pyodide";
+
+// this initiates pyodide
+languagePluginLoader;
```

Comment:
Would probably move this to a startup epic that matches how the jupyter kernel is initiated on app load